src/AvTranscoder/encoder/AudioEncoder.cpp

```diff
@@ -128,15 +128,15 @@ bool AudioEncoder::encode(const AVFrame* decodedData, AVPacket& encodedData)
         throw std::runtime_error("Encode audio frame error: avcodec encode audio frame - " +
                                  getDescriptionFromErrorCode(ret));
     }
-    return ret == 0 && gotPacket == 1;
+    return gotPacket == 1;
 #else
     const int ret = avcodec_encode_audio(&avCodecContext, encodedData.data, encodedData.size, decodedData);
     if(ret < 0)
     {
         throw std::runtime_error("Encode audio frame error: avcodec encode audio frame - " +
                                  getDescriptionFromErrorCode(ret));
     }
-    return ret == 0;
+    return true;
```

Comment:
Yes: https://ffmpeg.org/doxygen/2.7/group__lavc__encoding.html#gaf12a9da0d33f50ff406e03572fab4763
"""
Returns:
On error a negative value is returned, on success zero or the number of bytes used to encode the data read from the input buffer.
"""